Update creating a new connector page (#416)
wso2/docs-mi: en/docs/develop/customizations/creating-new-connector.md

> ### Step 1: Adding the connector to your mediation sequence

Reviewer: ```suggestion
### Step 1: Add the connector to your mediation sequence
```

1. [Set up WSO2 Micro Integrator Visual Studio Code extension]({{base_path}}/develop/installing-wso2-integration-studio).
2. [Create an ESB Config project]({{base_path}}/develop/create-integration-project) and add the connector ZIP file to the directory `/src/main/wso2mi/resources/connectors/` of your project.

    !!! Tip
        Be sure to select the new `googleBooks-connector-1.0.0.zip` file from your `org.wso2.carbon.esb.connector.googlebooks/target` directory.

> 3. [Create a custom proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service) named **googlebooks_listVolume**. In the **Graphical View**, you will see that the new connector is added to the tool palette under the **Local Connectors** section.

Reviewer: ```suggestion
3. [Create a custom proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service) named **googlebooks_listVolume**. In the **Graphical View**, you will see that the new connector has been added to the tool palette under the **Local Connectors** section.
```

             <respond/>
          </inSequence>
       </target>
    </proxy>
    ```

> ### Step 2: Building and deploying the artifacts

Reviewer: ```suggestion
### Step 2: Build and deploy the artifacts
```

You can click on the **Build and Run** option to build the Composite Application project and deploy the artifacts in the Micro Integrator. Alternatively, you can use the **Build** and **Export** options to build the artifacts and export them as a CAR file.

> ### Step 3: Testing the connector

Reviewer: ```suggestion
### Step 3: Test the connector
```
